src/modules/Search/SearchCategory.d.ts

```diff
@@ -29,14 +30,10 @@ export interface StrictSearchCategoryProps {
   /**
    * Renders the SearchCategory layout.
    *
-   * @param {object} categoryContent - The Renderable SearchCategory contents.
-   * @param {object} resultsContent - The Renderable SearchResult contents.
+   * @param {object} props - The SearchCategoryLayout props object.
    * @returns {*} - Renderable SearchCategory layout.
    */
-  layoutRenderer?: (
-    categoryContent: React.ReactElement<any>,
-    resultsContent: React.ReactElement<any>,
-  ) => React.ReactElement<any>
+  layoutRenderer?: (props: SearchCategoryLayoutProps) => React.ReactElement<any>
```

Comment:
```suggestion
  layoutRenderer?: (props: Pick<SearchCategoryLayoutProps, 'categoryContent' | 'resultsContent'>) => React.ReactElement<any>
```

We don't pass all props there only `categoryContent` & `resultsContent`, so let's make it more accurate 😉

Reply:
You're right! Thank you! 😀